src/platform_impl/apple/uikit/window.rs

```diff
@@ -365,12 +365,20 @@ impl Inner {
         warn!("`Window::set_ime_cursor_area` is ignored on iOS")
     }
 
-    pub fn set_ime_allowed(&self, _allowed: bool) {
-        warn!("`Window::set_ime_allowed` is ignored on iOS")
+    pub fn set_ime_allowed(&self, allowed: bool) {
+        if allowed {
+            unsafe {
+                self.view.becomeFirstResponder();
+            }
+        } else {
+            unsafe {
+                self.view.resignFirstResponder();
+            }
+        }
```

Comment:
This is what actually opens the keyboard. `becomeFirstResponder` requests focus for the winit view, and since it implements UIKeyInput iOS will now show the keyboard. `resignFirstResponder` resigns focus and hides the keyboard. I'll add a comment to explain this.

Not sure if the winit view not being first responder will ever be a problem, but I don't think it will, since before this PR the winit view never was first responder (the default of `canBecomeFirstResponder` is false)